Misc/NEWS.d/next/Library/2022-06-30-21-28-41.gh-issue-94440.LtgX0d.rst

```diff
@@ -0,0 +1,3 @@
+Fixed ``concurrent.futures.process._ExecutorManagerThread`` to flush canceled

```

Comment:
It's best not to discuss private, internal classes like `_ExecutorManagerThread` in news entries -- users of `concurrent.futures` generally won't care about such things. Try to frame the news entry in terms of the user-visible impact of the change instead.

Reply:
Makes sense - I rewrote the news item in that spirit.

Reply:
Thanks! It looks like the latest commit has been authored by a GitHub account associated with a different email address. That's caused the CLA check to start to fail -- you may need to sign it with both email addresses in order to appease the bot :)